blueocean-rest-impl/pom.xml

```diff
@@ -16,6 +16,8 @@
 
     <properties>
         <jacoco.haltOnFailure>true</jacoco.haltOnFailure>
+        <!-- Skip DefaultRunImpl for test coverage for now-->
+        <jacoco.missedclass.coverage>1.00</jacoco.missedclass.coverage>
```

Comment:
@imeredith did you update latest master, there is `DefaultRunImplTest` now so this exclusion is not needed. https://github.com/jenkinsci/blueocean-plugin/blob/15fff7ac07deb5b2a5eebbbdf941416f6e12f0da/blueocean-rest-impl/src/test/java/io/jenkins/blueocean/service/embedded/rest/DefaultRunImplTest.java#L16